src/Files.App/App.xaml

```diff
@@ -29,6 +29,7 @@
 			<ResourceDictionary.MergedDictionaries>
 				<XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
 				<ResourceDictionary Source="/ResourceDictionaries/PathIcons.xaml" />
+				<ResourceDictionary Source="/UserControls/Settings/SettingsControls.xaml" />
```

Comment:
It might be better to merge this on a per page basis.

Reply:
Each settings page and properties page?

Reply:
Fixed